dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -0,0 +1,468 @@
+					? 'horizontal'
+					: 'vertical'
+			}
+			imagePosition="left"
+			imagePositionOnMobile="top"
+			imageSize="large"
+			headlineSize="large"
+			headlineSizeOnMobile="large"
+		/>
+	);
+};
+
+/**
+ * ._________.
+ * |#########|
+ * |#########|
+ * |_________|
+ * Card designed to take up 25% of the container, with media that takes up 25%
+ *
+ * Options:
+ *  - Medium headline (medium on mobile)
+ *  - Small image on the top (left on mobile)
+ *  - No trail text
+ *  - Up to 2 supporting content items, always aligned vertical
+ */
+export const Card25Media25 = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			supportingContent={trail.supportingContent?.slice(0, 2)}
+			supportingContentAlignment="vertical"
+			containerPalette={containerPalette}
+			showAge={showAge}
+			imagePosition="top"
+			imagePositionOnMobile="left"
+			imageSize="small"
+			headlineSize="medium"
+			headlineSizeOnMobile="medium"
+		/>
+	);
+};
+/**
+ * ._________.
+ * |#########|
+ * |#########|
+ * |         |
+ * |_________|
+ * Card designed to take up 25% of the container, with media that takes up 25%
+ *
+ * Options:
+ *  - Medium headline (medium on mobile)
+ *  - Small image on the top (left on mobile)
+ *  - Trail text when there is no supporting content
+ *  - Up to 2 supporting content items, always aligned vertical
+ */
+export const Card25Media25Tall = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			showAge={showAge}
+			imagePosition="top"
+			imagePositionOnMobile="left"
+			imageSize="small"
+			headlineSize="medium"
+			headlineSizeOnMobile="medium"
+			trailText={
+				trail.supportingContent === undefined ||
+				trail.supportingContent.length === 0
+					? trail.trailText
+					: undefined
+			}
+			supportingContent={trail.supportingContent?.slice(0, 2)}
+		/>
+	);
+};
+
+/**
+ * .__________________.
+ * |##################|
+ * |##################|
+ * |__________________|
+ * Card designed to take up 50% of the container, with media that takes up 50%
+ *
+ * Options:
+ *  - Large headline (large on mobile)
+ *  - Small image on the top (top on mobile)
+ *  - No trail text
+ *  - Up to 3 supporting content items, always aligned horizontal
+ */
+export const Card50Media50 = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			headlineSize="large"
+			headlineSizeOnMobile="large"
+			imageSize="small"
+			imagePosition="top"
+			imagePositionOnMobile="top"
+			showAge={showAge}
+			supportingContent={trail.supportingContent?.slice(0, 3)}
+			supportingContentAlignment={'horizontal'}
+		/>
+	);
+};
+
+/**
+ * .__________________.
+ * |##################|
+ * |##################|
+ * |                  |
+ * |__________________|
```

Comment:
Just a thought [box drawing](https://en.wikipedia.org/wiki/Box-drawing_character)

```suggestion
 * ┏━━━━━━━━━━━━━━━━━━┱┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┐
 * ┃▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒┃                  ┊
 * ┃▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒┃                  ┊
 * ┃                  ┃                  ┊
 * ┗━━━━━━━━━━━━━━━━━━┹┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┘
```

Reply:
Good idea! Refactor incoming